### ISSUE TITLE
Map creation does not work

### ISSUE THREAD

**Issue by kaustubh-nair** [publiclab/mapknitter#759]
I tried to create a map locally, and I get this error 
```
 Showing /home/k/proj/mapknitter/app/views/layouts/knitter2.html.erb where line #85 raised:

undefined method `image' for nil:NilClass
```
I solved it by removing this commit 5f59c7eb58c5d0e406b84ccc26b9c70ae35bc79a

**Comment by kaustubh-nair** [comment]
@divyabaid16 Can you have a look at this?
Thanks!

**Comment by divyabaid16** [comment]
Ya, I will have a look at it.

**Comment by cesswairimu** [comment]
oh I fixed in #762. It should be fine now

**Comment by cesswairimu** [comment]
Closing this, thanks all
